basics/slices.md

```diff
@@ -0,0 +1,87 @@
+# Slices
+
+Slices sind Objekte des Types `T[]` für einen beliebigen Typen `T`.
+Slices sind eine Untermenge (Sicht) oder die gesamte Menge
+eines Arrays.
+**Dynamische Arrays sind Slices.**
+
+Ein Slice beinhaltet zwei Eigenschaften: (1) einen Zeiger auf das erste Element und
+(2) die Länge des Slices:
+
+    T* ptr;
+    size_t length; // uint auf 32-bit, ulong auf 64-bit (Siehe http://tour.dlang.org/tour/en/basics/basic-types)
```

Comment:
Hmm, ich glaube es ist besser wenn wir den Link in den plaintext-Teil packen, s.d. er anklickbar ist. Außerdem linkt er aktuell zur englischen Version, ist HTTP (der PR für HTTPS always wurde schon gemergt, nur noch nicht deploy), und würde Benutzer von tour.dlang.io oder der lokalen Version unnötig umleiten.

tl;dr:  für Links ist `[basics](basics/basic-types)` am Besten geeignet.

Ein erster Vorschlag:

> `size_t` entspricht dem Datentyp der maximalen Arraygröße der Zielplatform z.B. `uint` auf 32-bit Architekturen und `ulong` auf 64-bit Architekturen (für detaillierte Informationen `[gehe zurück in das Datentypen-Kapitel](basics/basics-types)`).